Branch-SDK/src/io/branch/referral/BranchConfig.java

```diff
@@ -0,0 +1,8 @@
+package io.branch.referral;
+
+/*
+ *
+ */
+public class BranchConfig {
```

Comment:
Do you have plan to move more stuff into this class? If only for this TAG, maybe we can just put it into PrefHelper.